MdePkg/Library/UefiUsbLib/UsbDxeLib.c

```diff
@@ -4,12 +4,16 @@
   in Usb specification 9.4 section.
 
   Copyright (c) 2004 - 2018, Intel Corporation. All rights reserved.<BR>
+  Copyright (c) 2024, American Megatrends International LLC. All rights reserved.<BR>
   SPDX-License-Identifier: BSD-2-Clause-Patent
 
 **/
 
 #include "UefiUsbLibInternal.h"
 
+static UINT8                      *mConfigData;

```

Comment:
Set to NULL and remove constructor function from this file and INF

Reply:
done